alembic/versions/37c8b61f23ce_upd_temperature.py

```diff
@@ -0,0 +1,30 @@
+"""upd temperature
+
+Revision ID: 37c8b61f23ce
+Revises: 2158aff077f4
+Create Date: 2024-04-21 01:26:58.557446
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '37c8b61f23ce'
+down_revision: Union[str, None] = '2158aff077f4'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `upgrade` function is currently empty. If this migration is supposed to make changes to the database schema, you should add the necessary Alembic operations here.